### PR TITLE
Feature/angular

### PR DESCRIPTION
**Pull request by CapnCrunchsna**
Very many changes and additions.  Angularizing everything and adding features.  Future branches will be much smaller.

### CODE REVIEW

File: public/angular/about/about.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+angular.module('fc.about', [])
+.controller('AboutCtrl', function($scope) {
```
Comment (marshall-davis) [inline public/angular/about/about.js:2]: You don't indent multi-line chains?

File: public/angular/app.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+angular.module('fc', [
+   'ui.router', 
+   'fc.nav',
+   'fc.home',
+   'fc.comingSoon',
+   'fc.account.register'
+]).config(function($stateProvider, $urlRouterProvider) {
```
Comment (marshall-davis) [inline public/angular/app.js:7]: In other instances you dropped the chained methods down a line, here you did not.

File: public/angular/bower.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "<none>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/FastingCrusade/website",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
```
Comment (marshall-davis) [inline public/angular/bower.json:15]: If this is actual testing, I think it should be committed. One of things I'm going to do is start running CI tests against the back-end now that @trumpsnare is helping. We could run acceptance tests on Codeception or something too.

File: resources/views/soon.blade.php [hunk 4]
```diff
@@ -25,7 +25,7 @@
     </div>
     <div class="row">
         <div class="column">
-            Your site for fasting.
+            Your site for fasting.  Also for testing.
```
Comment (marshall-davis) [inline resources/views/soon.blade.php:28]: Womp Womp  
Remember not to deploy this change to production early!

File: public/angular/about/about.js (hunk 1 above)
Comment (CapnCrunchsna) [reply public/angular/about/about.js:2]: I do.  Mostly.  I struggle with the tops of angular files, and we do at work too.  So I went with what we did there.  If you indent every multi-line chain with the angular controllers (and directives and such) then you just have lots and lots of indents that aren't really helpful.

File: public/angular/app.js (hunk 2 above)
Comment (CapnCrunchsna) [reply public/angular/app.js:7]: Yep, I can fix that.

File: public/angular/soon/coming-soon.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<div class="coming-soon">
+   <div class="row">
+      <div class="column-md-12 main-logo">
+         <img src="img/c_logo.svg" style="height: 10rem">
```
Comment (marshall-davis) [inline public/angular/soon/coming-soon.html:4]: Look at that! A `.svg`! Y U NO ALWAYS?

File: public/angular/bower.json (hunk 3 above)
Comment (CapnCrunchsna) [reply public/angular/bower.json:15]: This is something bower made on init.  I'm not actually sure what exactly it is ignoring, but if we do eventually do actual tests they should certainly be committed.

File: resources/views/soon.blade.php (hunk 4 above)
Comment (CapnCrunchsna) [reply resources/views/soon.blade.php:28]: Hmm, there may be more of these...

File: public/angular/soon/coming-soon.html (hunk 5 above)
Comment (CapnCrunchsna) [reply public/angular/soon/coming-soon.html:4]: Alwaysing is hard!

**Pull request merged**